<?xml version="1.0"?>
<launch>
    <arg name="run_urg"          default="true"/>
    <arg name="min_angle"        default="-45"/>
    <arg name="max_angle"        default="45"/>
    <arg name="laser_topic_in"   default="/bender/sensors/hokuyo_front/scan"/>
    <arg name="laser_topic_out"  default="/bender/sensors/hokuyo_front/scan_cropped"/>
    
    <include file="$(find urg_node)/launch/urg_lidar.launch" if="$(arg run_urg)"/>
    
    <node name="laser_cropper_node" pkg="uchile_laser_pipeline" type="laser_cropper"  output="screen" respawn="true">
        <param name="camera_fov_start_angle" value="$(arg min_angle)" />
        <param name="camera_fov_end_angle"   value="$(arg max_angle)" />
        <param name="laser_topic_in"         value="$(arg laser_topic_in)" />
        <param name="laser_topic_out"        value="$(arg laser_topic_out)" />
    </node>
</launch>
